lib/shoulda/matchers/action_controller/permit_matcher.rb

```diff
@@ -215,7 +215,7 @@ def set_double_collection
           @double_collection =
             Doublespeak.double_collection_for(::ActionController::Parameters)
 
-          @double_collection.register_stub(:require).to_return { |params| params }
+          @double_collection.register_stub(:require).to_return { |call| call.object }
```

Comment:
Line is too long. [85/80]<br>Pass `&:object` as an argument to `to_return` instead of a block.